packages/docusaurus-theme-classic/codeTranslations/fa.json

```diff
@@ -1,56 +1,56 @@
   "theme.blog.paginator.olderEntries": "مطالب قدیمی تر",
   "theme.blog.post.paginator.navAriaLabel": "کنترل پست های صفحه وبلاگ",
-  "theme.blog.post.paginator.newerPost": "پست های جدید تر",
-  "theme.blog.post.paginator.olderPost": "پست های قدیمی تر",
-  "theme.blog.post.plurals": "One post|{count} posts",
+  "theme.blog.post.paginator.newerPost": "پست جدید تر",
+  "theme.blog.post.paginator.olderPost": "پست قدیمی تر",
+  "theme.blog.post.plurals": "یک پست|{count} پست",
   "theme.blog.post.readMore": "ادامه مطلب",
-  "theme.blog.post.readingTime.plurals": "خواندن در {readingTime} دقیقه|خواندن در {readingTime} دقیقه",
-  "theme.blog.sidebar.navAriaLabel": "Blog recent posts navigation",
+  "theme.blog.post.readingTime.plurals": "خواندن ۱ دقیقه|خواندن {readingTime} دقیقه",
+  "theme.blog.sidebar.navAriaLabel": "کنترل پست های اخیر وبلاگ",
   "theme.blog.tagTitle": "{nPosts} با برچسب \"{tagName}\"",
-  "theme.common.editThisPage": "ویرایش صفحه",
+  "theme.common.editThisPage": "ویرایش مطالب این صفحه",
   "theme.common.headingLinkTitle": "لینک مستقیم به عنوان",
-  "theme.common.skipToMainContent": "رفتن به مطلب اصلی",
-  "theme.docs.paginator.navAriaLabel": "کنترل صفحه اسناد",
+  "theme.common.skipToMainContent": "پرش به مطلب اصلی",
+  "theme.docs.paginator.navAriaLabel": "کنترل صفحات مطالب",
   "theme.docs.paginator.next": "بعدی",
   "theme.docs.paginator.previous": "قبلی",
   "theme.docs.sidebar.collapseButtonAriaLabel": "بستن نوار کناری",
   "theme.docs.sidebar.collapseButtonTitle": "بستن نوار کناری",
-  "theme.docs.sidebar.expandButtonAriaLabel": "بزرگ کردن نوار کناری",
-  "theme.docs.sidebar.expandButtonTitle": "بزرگ کردن نوار کناری",
+  "theme.docs.sidebar.expandButtonAriaLabel": "باز کردن نوار کناری",
+  "theme.docs.sidebar.expandButtonTitle": "باز کردن نوار کناری",
   "theme.docs.versions.latestVersionLinkLabel": "آخرین نسخه",
-  "theme.docs.versions.latestVersionSuggestionLabel": "برای دیدن آخرین نسخه، {latestVersionLink} ({versionLabel}) را ببینید.",
-  "theme.docs.versions.unmaintainedVersionLabel": "نسخه {siteTitle} {versionLabel} دیگر بروزرسانی نمی شود.",
+  "theme.docs.versions.latestVersionSuggestionLabel": "برای دیدن آخرین نسخه این متن، نسخه {latestVersionLink} ({versionLabel}) را ببینید.",
+  "theme.docs.versions.unmaintainedVersionLabel": "نسخه {siteTitle} {versionLabel} دیگر به روزرسانی نمی شود.",
   "theme.docs.versions.unreleasedVersionLabel": "نسخه {siteTitle} {versionLabel} هنوز به صورت کامل منتشر نشده است.",
   "theme.lastUpdated.atDate": " در تاریخ {date}",
   "theme.lastUpdated.byUser": " توسط {user}",
-  "theme.lastUpdated.lastUpdatedAtBy": "آخرین به روزرسانی{atDate}{byUser}",
-  "theme.navbar.mobileSidebarSecondaryMenu.backButtonLabel": "← Back to main menu",
-  "theme.tags.tagsListLabel": ":برچسب ها",
+  "theme.lastUpdated.lastUpdatedAtBy": "آخرین بروزرسانی در {atDate} توسط {byUser}",
+  "theme.navbar.mobileSidebarSecondaryMenu.backButtonLabel": "← بازگشت به منو اصلی",
```

Comment:
going to merge this now, but please double-check that you don't need to "flip" the arrow?

![image](https://user-images.githubusercontent.com/749374/127125214-8899bf2b-6d8e-449b-af5e-23eb6d9a2e79.png)